ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -837,9 +837,9 @@ Handle Deletion Confirmation Modal
 Click Action From Actions Menu
     [Documentation]    Clicks an action from Actions menu (3-dots menu on the right)
     [Arguments]    ${item_title}    ${action}    ${item_type}=${NONE}
-    Click Element       xpath=//tr[td[@data-label="Name"]//*[text()="${item_title}"]]/td[@class="pf-v5-c-table__action"]/div/button[@aria-label="Actions"]
+    Click Element       xpath=//tr[td[@data-label="Name"]//*[text()="${item_title}"]]/td[contains(@class,"-table__action")]//button[@aria-label="Kebab toggle"]    # robocop: disable
```

Comment:
Note to self, for the future we should have keywords for clicking common paternfly UI elements. Locators of this style appear many times. `@aria-label="Kebab toggle"` is beginning to hound my dreams.

Reply:
yep I agree, that would help also reducing the code duplication. However I don't kow now from top of my head some more keywords like this one we could have. But if you find any, welcome to have it in a common place to be re-used